### DISCUSSION TITLE
关于训练一个AI的想法

### DISCUSSION THREAD

**Discussion by Yeuoly** [ba-archive/blue-archive#205] [Ideas]
我想要通过该repo的数据去训练一个BA的GPT3.5 微调模型，每个学生都可以进行针对性训练，得到一个赛博BA模拟器，希望得到项目方的授权，谢谢！

**Comment by mark9804** [comment]
我们内部也有 LLM 开发者在考虑做这个模型，我们考虑用 Qwen-7b，有兴趣的话可以加个后端群互相分享信息吗？

`*********`
